pkg/apis/core/types.go

```diff
@@ -3076,6 +3078,12 @@ type PodSecurityContext struct {
 	// +k8s:conversion-gen=false
 	// +optional
 	ShareProcessNamespace *bool
+	// Use the host's user namespace.
+	// Optional: Default to true.
```

Comment:
Default to `true`?

Reply:
yes, the default is not change the current behavior where all pods are running in the host user namespace

Reply:
I know why we are doing this, but is there any path to "safe by default", short of core/v2 ?

Reply:
I propose that we start thinking about core/v2 after this goes GA - start accumulating the set of things that we are willing to change (like default values) to make a saner "by default" experience.  I can already think of a bunch :)